### DISCUSSION TITLE
OpenMS 3.0.0

### DISCUSSION THREAD

**Discussion by jpfeuffer** [OpenMS/OpenMS#6954] [General]
Dear OpenMS-Users,

we are proud to announce the release of OpenMS 3.0.0. Grab it
<a href="https://abibuilder.cs.uni-tuebingen.de/archive/openms/OpenMSInstaller/release/3.0.0">here</a>

In the following you find all important changes to the previous version:

------------------------------------------------------------------------------------------

- full release
- OpenSwath: Add support for diaPASEF data with overlapping m/z and IM windows, and add new outputs on ion mobility features (delta_im)
- OpenSwath: Fix duplicated transition error when multiple genes map to a single peptide (#5653)
- TOPPView: TheoreticalSpectrumGenerationDialog now supports generation of isotope patterns for metabolites
- pyopenms: pyopenms-extra is renamed to pyopenms-docs.
- Add extractSpectra to TargetedSpectraExtractor with support of MS1 features
- TargetedSpectraExtractor::searchSpectrum added an option to add unknown features to the result featureMap
- Fixed race condition when logging messages.
- Added the support for 'no cleavage' for XTandemAdapter and CometAdapter (#6133).
- ParamEditor with more convenient StringList editing  (#5135)
- TOPPAS: add a `recent files` submenu
- removed InspectAdapter
- removed OMSSAAdapter
- removed MyriMatchAdapter
- removed Crux
- OpenMS documentation is moved to openms.readthedocs.io/en/latest. OpenMS API
reference and advanced developer documentation remains inside OpenMS doxygen
documentation.
- Add support for user defined ribonucleotides with phosphorothioate linkages
- Add support for JSON based ribonucleotides and updated to latest Modomics database
- fix bug in GaussFilter when using ppm as width (#6830)
- Add support to load parameter values from a JSON formatted .json file. (Accessible via -ini. This will be
  helpful for Common Workflow Language users and others)

New Tools:
- FLASHDeconv -- Ultra-fast high-quality deconvolution enables online processing of top-down MS data (TOPP)
- FLASHDeconvWizard -- An GUI assistant for FLASHDeconv execution.

------------------------------------------------------------------------------------------

Best regards,
The OpenMS-Developers

<hr /><em>This discussion was created from the release <a href='https://github.com/OpenMS/OpenMS/releases/tag/Release3.0.0'>OpenMS 3.0.0</a>.</em>

**Comment by radusuciu** [comment]
Congrats on the release!
